docs/contribution.md

```diff
@@ -96,6 +97,81 @@ Success: no issues found in 6 source files
   congratulations :) (28.89 seconds)
 ```
 
+## Unit tests
+
+To keep high quality code, we require to provide a Pytest for every tests implemented in ANTA.
+
+All submodule should have its own pytest section under `tests/units/anta_tests/<submodule-name>`. In this directory, you should have 3 files:
+
+- `__init__.py`: Just because it is used as a python module
+- `data.py`: Where all your parametrize go. So all your test information should be located here
+- `test_exc.py`: Pytest file with test definition.
+
+A pytest definition should be similar to this template:
+
+```python
+# -*- coding: utf-8 -*-
+
+"""
+Tests for anta.tests.hardware.py
+"""
+from __future__ import annotations
+
+import asyncio
+import logging
+from typing import Any
+from unittest.mock import MagicMock
+
+import pytest
+
+from anta.tests.hardware import VerifyAdverseDrops
+from tests.lib.utils import generate_test_ids_list
+
+from .data import INPUT_<TEST_NAME>
+
+@pytest.mark.parametrize("test_data", INPUT_<TEST_NAME>, ids=generate_test_ids_list(INPUT_<TEST_NAME>))
+def test_<TEST_CASE>(mocked_device: MagicMock, test_data: Any) -> None:
+    """Check <TEST_CASE>."""
+
+    logging.info(f"Mocked device is: {mocked_device.host}")
+    logging.info(f"Mocked HW is: {mocked_device.hw_model}")
+
+    test = <TEST_CASE>(mocked_device, eos_data=test_data["eos_data"])
+    asyncio.run(test.test())
+
+    logging.debug(f"test result is: {test.result}")
+
+    assert str(test.result.name) == mocked_device.name
+    assert test.result.result == test_data["expected_result"]
+```
+
+The `mocked_device` object is a fixture defined in Pytest to represent an InventoryDevice and the parametrize `test_data` is a list of dictionries with structure:
+
+```json
+INPUT_RUNNING_CONFIG: List[Dict[str, Any]] = [
+// Test Case #1
```

Comment:
is this a correct syntax? maybe use # instead and indicate that it is in python?